domain/src/main/java/me/androidbox/domain/alarm_manager/AlarmItem.kt

```diff
@@ -1,13 +1,27 @@
 package me.androidbox.domain.alarm_manager
 
-import java.time.ZonedDateTime
-
+import me.androidbox.domain.authentication.model.Event
 
 data class AlarmItem(
-    val dateTime: ZonedDateTime,
-    val message: String?
-) {
-    val long = dateTime.toEpochSecond() * 100
-
+    val agendaId: String,
+    val title: String,
+    val description: String,
+    val remindAt: Long,
+    val agendaType: AgendaType
+)
 
+fun Event.toAlarmItem(agendaType: AgendaType): AlarmItem {
+    return AlarmItem(
+        agendaId = this.id,
+        title = this.title,
+        description = this.description,
+        remindAt = this.remindAt * 1000,
```

Comment:
Why not have one consistent scale for timestamps? Nothing tells me here that `remindAt` for the event is in seconds and for the alarm item in milliseconds. Things like these will 99% lead to a bug especially when working together with other devs.
So I'd just default all timestamps in the app to be in milliseconds or seconds